apps/ledger-live-mobile/src/screens/Settings/Debug/Generators/index.tsx

```diff
@@ -29,6 +102,34 @@ export default function Generators() {
         title="Import .env BRIDGESTREAM_DATA"
         dataStr={config.BRIDGESTREAM_DATA}
       />
+      <Alert
+        type="error"
+        title="The rows below perform destructive operations and should only be used if you know exactly what you're doing."
+      />
+      <SettingsRow
+        title="Refresh?"
+        desc="You may want to reload the app after wiping data"
+        iconLeft={<Icons.RefreshMedium size={24} color="black" />}
+        onPress={onForceRefresh}
+      />
+      <SettingsRow
+        title="Settings"
+        desc="Restores all settings to their default values"
```

Comment:
Maybe use imperative here "Restore all settings" like in other descriptions

Reply:
Suggest the edit here and we can merge it